gateway/api_test.go

```diff
@@ -84,19 +84,28 @@ func TestPolicyAPI(t *testing.T) {
 
 	defer ts.Close()
 	ts.Gw.BuildAndLoadAPI()
+	ts.Gw.DoReload()
```

Comment:
I think in the same `Run` like the left, `reload` should be called with API call as a real flow.